crates/ra_syntax/src/yellow/green.rs

```diff
@@ -2,7 +2,7 @@ use std::sync::Arc;
 
 use smol_str::SmolStr;
 
-use {SyntaxKind, TextUnit};
+use crate::{SyntaxKind, TextUnit};
```

Comment:
Let's try to avoid this in this PR? I have a significant refactor in progress (https://github.com/matklad/rust-analyzer/pull/85), so it's better to avoid merge conflicts if possible :)

Reply:
ok, revert it soon(`cargo fix` since `cargo test` failed locally)